From ae5a255001f17b35b510a3d70fbf5e770acb1dd9 Mon Sep 17 00:00:00 2001
From: Leonardo Graboski Veiga <leonardo.veiga@toradex.com>
Date: Wed, 6 Nov 2019 09:19:16 -0300
Subject: [PATCH 1/2] pasta-demo: support wayland weston, vulkan, etc

We may need to run graphics outside containers. This is due to timeline
restrictions and it should be possible to run everything in a container
in the future.
---
 recipes-graphics/imx-gpu-viv/imx-gpu-viv-6.inc | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/recipes-graphics/imx-gpu-viv/imx-gpu-viv-6.inc b/recipes-graphics/imx-gpu-viv/imx-gpu-viv-6.inc
index 2c61c101..030f1ea7 100644
--- a/recipes-graphics/imx-gpu-viv/imx-gpu-viv-6.inc
+++ b/recipes-graphics/imx-gpu-viv/imx-gpu-viv-6.inc
@@ -33,6 +33,7 @@ EXTRA_PROVIDES_append_imxgpu3d = " \
 "
 EXTRA_PROVIDES_append_mx8 = " \
     virtual/libgbm \
+    vulkan \
 "
 EXTRA_PROVIDES_append_mx8qm = " \
     virtual/libopenvx \
@@ -368,4 +369,4 @@ FILES_imx-gpu-viv-tools = "${bindir}/gmem_info"
 FILES_imx-gpu-viv-demos = "/opt"
 INSANE_SKIP_imx-gpu-viv-demos += "rpaths dev-deps"
 
-# COMPATIBLE_MACHINE = "(mx6q|mx6dl|mx6sx|mx6sl|mx8)"
+COMPATIBLE_MACHINE = "(mx6q|mx6dl|mx6sx|mx6sl|mx8|apalis-imx8)"
-- 
2.17.1

